auth-middleware-test-svc/flaskapp.py

```diff
@@ -12,8 +12,20 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# use this for installed auth library
 import flask
 import auth_middleware
+from dotenv import load_dotenv
+
+# use this for development without installing auth library
```

Comment:
```
# use this for development without installing auth library
# changed the name lib/auth-middleware to lib/auth_middleware.
# Change it back when push
# import flask
# import lib.auth_middleware.auth_middleware as auth_middleware
# from dotenv import load_dotenv
```
Are these comments needed?

Reply:
just left it for the future development. Removed.

Reply:
Thanks. I think it'll be better to implement code changes without changing the way the library is installed. That may create some confusions.